app/src/main/java/com/andrew/tasky/presentation/agenda/AgendaFragment.kt

```diff
@@ -1,64 +1,279 @@
+        val todayDate = currentDateAndTime.toLocalDate()
+        val tomorrowDate = currentDateAndTime.toLocalDate().plusDays(1)
+        val thirdDayDate = currentDateAndTime.toLocalDate().plusDays(2)
+        val fourthDayDate = currentDateAndTime.toLocalDate().plusDays(3)
+        val fifthDayDate = currentDateAndTime.toLocalDate().plusDays(4)
+        val sixthDayDate = currentDateAndTime.toLocalDate().plusDays(5)
+
+        //sets up dateNumber text view in the mini calendar
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.today).dateNumber.text =
+            todayDate.format(DateTimeFormatter.ofPattern("d"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.tomorrow).dateNumber.text =
+            tomorrowDate.format(DateTimeFormatter.ofPattern("d"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.thirdDay).dateNumber.text =
+            thirdDayDate.format(DateTimeFormatter.ofPattern("d"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.fourthDay).dateNumber.text =
+            fourthDayDate.format(DateTimeFormatter.ofPattern("d"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.fifthDay).dateNumber.text =
+            fifthDayDate.format(DateTimeFormatter.ofPattern("d"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.sixthDay).dateNumber.text =
+            sixthDayDate.format(DateTimeFormatter.ofPattern("d"))
 
-        binding.addReminder.setOnClickListener(){
-            val agendaItemType = AgendaItemType.REMINDER.name
-            navController.navigate(AgendaFragmentDirections
-                .actionAgendaFragmentToAgendaItemDetailFragment(agendaItemType))
+        //sets up dayOfWeek text view in mini calendar
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.today).dayOfWeek.text =
+            todayDate.format(DateTimeFormatter.ofPattern("eeeee"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.tomorrow).dayOfWeek.text =
+            tomorrowDate.format(DateTimeFormatter.ofPattern("eeeee"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.thirdDay).dayOfWeek.text =
+            thirdDayDate.format(DateTimeFormatter.ofPattern("eeeee"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.fourthDay).dayOfWeek.text =
+            fourthDayDate.format(DateTimeFormatter.ofPattern("eeeee"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.fifthDay).dayOfWeek.text =
+            fifthDayDate.format(DateTimeFormatter.ofPattern("eeeee"))
+        ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.sixthDay).dayOfWeek.text =
+            sixthDayDate.format(DateTimeFormatter.ofPattern("eeeee"))
+
+        //If dateSelected is on miniCalendar, it sets the background color to yellow
+        if (dateSelected == todayDate) {
+            ItemMiniCalendarDayBinding.bind(fragmentAgendaBinding.today).miniCalendarCard
+                .setCardBackgroundColor(Color.parseColor("#fdeda7"))
```

Comment:
I'd avoid using hardcoded colors here. If that requirement changes, it'll become a headache to change them all. Instead, I'd recommend to use color resources (from an XML file)